client/app/app.js

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'preact'
```

Comment:
Sorry for this double review but I didn't noticed it before. I think we should use `.jsx` extension for files handling JSX code, it's better for linting and highlighting when we develop using this language.